workflows/devnet-build/Jenkinsfile

```diff
@@ -21,13 +22,16 @@ pipeline{
                     }
                     buildEnv.inside {
                         dir('lisk'){
-                            git url: 'https://github.com/LiskHQ/lisk', branch: "${params.branch}"
+                            checkout([$class: 'GitSCM', branches: [[name: "${params.branch}" ]],
```

Comment:
Why is this change necessary?

Reply:
Because with the previous syntax you can specify a branch, but not a SHA or a tag.